igvc_navigation/launch/navigation_client.launch

```diff
@@ -1,6 +1,6 @@
 <launch>
+    <arg name="reading_from_file" default="true"/>
     <node name="navigation_client" pkg="igvc_navigation" type="navigation_client" output="screen">
-        <param name="reading_from_file" value="true"/>
```

Comment:
You still need this param here. It's just that the value is now `$(arg reading_from_file)` instead of being `true`